packages/eslint-config-next/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@next/eslint-plugin-next": "13.4.11-canary.0",
     "@rushstack/eslint-patch": "^1.1.3",
-    "@typescript-eslint/parser": "^5.42.0",
+    "@typescript-eslint/parser": "^6.0.0",
```

Comment:
Maybe `^5.4.2 || ^6.0.0`, so folks still on 5.x will still be supported without duplicate versions?

Reply:
@JoshuaKGoldberg I think this is a good shout and speaks to what @styfle was raising. Pushed a new commit to address this.